cmd/porter/bundle.go

```diff
@@ -288,7 +291,7 @@ func buildBundlePublishCommand(p *porter.Porter) *cobra.Command {
 
 	f := cmd.Flags()
 	f.StringVarP(&opts.File, "file", "f", "", "Path to the Porter manifest. Defaults to `porter.yaml` in the current directory.")
-	f.StringVarP(&opts.Tag, "tag", "t", "", "Tag to apply to the CNAB bundle. Defaults to the `Tag` value currently in the Porter manifest.")
+	f.BoolVar(&opts.InsecureRegistry, "insecure", false, "Treat the registry as insecure.")
```

Comment:
Just a note that whatever we change the flag description to above, let's change this one as well.